src/components/Example.tsx

```diff
@@ -63,14 +63,16 @@ export const Example = ({
             }`}
             passHref
           >
-            <IconButton
-              as={"a"}
-              cursor={"pointer"}
-              icon={<ExternalLinkIcon />}
-              size={"sm"}
-              aria-label={"Open in Fullscreen"}
-              title={"Open in Fullscreen"}
-            />
+            <a target="_blank">
```

Comment:
I guess the `target="_blank"` can be added directly to the `IconButton` because the button itself is rendered as an anchor. Otherwise there will be rendered 2 anchors.

Reply:
Didn't notice that! Thanks @hauptrolle!